tests/providers/yandex/hooks/test_yandex.py

```diff
@@ -0,0 +1,84 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+import mock
```

Comment:
```suggestion
from unittest import mock
```

Reply:
@mik-laj  Fixed this, is there anything else required to merge this PR, please let me know.

Reply:
As mentioned before - https://github.com/apache/airflow/pull/9665#issuecomment-653931518, you still have to delete this file from the list of missing tests. This list is covered by unit test. We must have green CI to be able to merge this change.
```
tests/test_logging_config.py ........                                    [ 16%]
tests/test_project_structure.py .F

___________ TestProjectStructure.test_keep_missing_test_files_update ___________

self = <tests.test_project_structure.TestProjectStructure testMethod=test_keep_missing_test_files_update>

    def test_keep_missing_test_files_update(self):
        new_test_files = []
        for test_file in MISSING_TEST_FILES:
            if os.path.isfile(f"{ROOT_FOLDER}/{test_file}"):
                new_test_files.append(test_file)
        if new_test_files:
            new_files_text = '\n'.join(new_test_files)
            self.fail(
>               "You've added a test file currently listed as missing:\n"
                f"{new_files_text}"
                "\n"
                "Thank you very much.\n"
                "Can you remove it from the list of missing tests, please?"
            )
E           AssertionError: You've added a test file currently listed as missing:
E           tests/providers/yandex/hooks/test_yandex.py
E           Thank you very much.
E           Can you remove it from the list of missing tests, please?

tests/test_project_structure.py:117: AssertionError
```

Reply:
thanks will do that

Reply:
@mik-laj removed the file from the list and I see green CI. Thanks ready for merge.